core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,50 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:600px){
+ .oppia-navbar-brand-name{
+  max-width: 35px;
+  overflow: hidden;
+ }
+ .protractor-test-exploration-info-icon{
```

Comment:
Protractor test classes should not be used in CSS. If you need another class, add one.